packages/typedoc-plugin-appium/lib/converter/comment.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Utilities to derive comments from various sources
+ * @module
+ */
+
+import _ from 'lodash';
+import {Comment, CommentDisplayPart} from 'typedoc';
+import {AsyncMethodDeclarationReflection, KnownMethods} from './types';
+
+export const stats = _.fill(new Array(5), 0);
```

Comment:
this is for debugging to let me know which "comment finders" are actually being called, so I can remove the ones that don't get called